docs/pallas/gpu/pipelining.md

```diff
@@ -0,0 +1,325 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    main_language: python
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.4
+  kernelspec:
+    display_name: Python 3
+    name: python3
+---
+
+(pallas_mgpu_pipelining)=
```

Comment:
OOC is that some way to create links to docs?

Reply:
Yes you can refer back to this with {ref}`pallas_mgpu_pipelining`